lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -7005,6 +7036,7 @@ class DecomposeComplexOpsPass
     addPatternIfTargetOpIsIllegal<DecomposeAtenReshapeAsOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenIndexTensorOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenTriuOp>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenLinalgNormOp>(patterns);
```

Comment:
You also need to add it here: https://github.com/llvm/torch-mlir/blob/e85a2a87c5662c26e047a2d93d3ff6216cabdcec/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L367